ydb/library/yql/minikql/mkql_key_payload_value_lru_cache.h

```diff
@@ -0,0 +1,128 @@
+#pragma once
+
+#include <ydb/library/yql/public/udf/udf_value.h>
+#include <ydb/library/yql/minikql/mkql_node.h>
+#include <ydb/library/yql/minikql/mkql_type_builder.h>
+#include <util/generic/string.h>
+#include <unordered_map>
+#include <list>
+#include <chrono>
+
+namespace NKikimr::NMiniKQL {
+
+//Design notes:
+//  Commodity LRUCache with HashMap and DoubleLinkedList. Key(that is TUnboxedValue) is stored in both HashMap and List
+//  Lazy(postponed) TTL implementation. An entry is not deleted immediately when its TTL is expired, the deletion is postponed until:
+//   - when it is accessed via Get()
+//   - when it is not accessed for a long period of time and becomes the LRU. Then it is garbage collected in Tick()
+//  In order to store values in HashMap serializes Key(TUnboxedValue) to TString
```

Comment:
seems it's wrong. NUdf::TUnboxedValue is a key for HashMap

Reply:
obsolete comment, removed